src/ucs/sys/sock.h

```diff
@@ -183,9 +183,8 @@ int ucs_socket_max_conn();
  * @param [in]      err_cb          Error callback.
  * @param [in]      err_cb_arg      User's argument for the error callback.
  *
- * @return UCS_OK on success, UCS_ERR_CANCELED if connection closed,
- *         UCS_ERR_NO_PROGRESS if system call was interrupted or
- *         would block, UCS_ERR_IO_ERROR on failure.
+ * @return UCS_OK on success, UCS_ERR_NO_PROGRESS if system call was interrupted
+ *         or would block, UCS_ERR_IO_ERROR on failure.
```

Comment:
Can we add more meaningful statuses? For example, `UCS_ERR_NOT_CONNECTED` would be more clear.
BTW, also we will have something like `REMOTE_DISCONNECTED` soon,  can we somehow to detect the difference?